app/celer-sim/RunnerInput.hh

```diff
@@ -105,7 +105,7 @@ struct RunnerInput
     bool action_times{};
     bool merge_events{false};  //!< Run all events at once on a single stream
     bool default_stream{false};  //!< Launch all kernels on the default stream
-    bool warm_up{CELER_USE_DEVICE};  //!< Run a nullop step first
+    bool warm_up{false};  //!< Run a nullop step first
```

Comment:
Maybe we should start using `std::optional` for some of these. Our whole input thing needs a refactor anyway to reduce duplication across the two apps.

Reply:
That could be nice... but yes, definitely needs a refactor, so perhaps I'll wait for that.